packages/devkit/src/generators/format-files.ts

```diff
@@ -32,7 +32,7 @@ export async function formatFiles(tree: Tree): Promise<void> {
   await Promise.all(
     Array.from(files).map(async (file) => {
       const systemPath = path.join(tree.root, file.path);
-      let options: Prettier.Options = {
+      let options: any = {
```

Comment:
There's probably a different type we can use here. Can you try Prettier.FileInfoOptions?

Reply:
```
Error: packages/devkit/src/generators/format-files.ts:36:9 - error TS2322: Type '{ filepath: string; }' is not assignable to type 'FileInfoOptions'.
  Object literal may only specify known properties, and 'filepath' does not exist in type 'FileInfoOptions'.

36         filepath: systemPath,
```

Reply:
Ah, yes. Try `Prettier.Options & Prettier.FileInfoOptions`